.jshintrc

```diff
@@ -5,6 +5,7 @@
 	"jquery" : false,
 	"couch" : false,
 	"es5" : false,
+	"esnext": true,
```

Comment:
Since the rest of the test suite is in ES5, it might make sense to retain uniformity.